src/js/components/Reputation/index.jsx

```diff
@@ -26,11 +26,11 @@ class Reputation extends Component {
       let classes = 'average-rating py-1 px-3 rounded ';
       if (!Number.isInteger(reputation.averageCount)) {
         classes += 'bg-dark text-dark';
-      } else  if (reputation.averageCount >= 4) {
+      } else  if (reputation.averageCount >= 8) {
```

Comment:
We have a condition in the contracts to not allow ratings above 5, so probably most of these conditions will not trigger.

Reply:
Nvm. I see you convert it below to base10!  clever!

Reply:
Yeah, it,s not ideal, but that way, we don't have to modify the contract haha